service/README.md

```diff
@@ -39,7 +39,7 @@ This will start the API container exposing the application's port (set to `4000`
 In order to test if the application is up, you can call its health endpoint:
 
 ```bash
-  curl http://localhost:4000/health
+  curl http://fpl-service:4000/health
```

Comment:
We have to revert it back to 

```suggestion
  curl http://localhost:4000/health
```

This call is meant to be made from host machine so `fpl-service` won't be recognised unless added to `/etc/hosts`